Watchman.Common/FriendlyDictionary.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Watchman.Common
+{
+    public class FriendlyDictionary<T, V>
```

Comment:
w czym ma nam pomóc ta klasa?

Reply:
Można zagnieżdzać do woli FDict<key, FDict<key, FDict<key, value>>> i nie trzeba na każdym poziomie sprawdzać czy dana wartość istnieje itp itd tylko od razu lecieć dict[a][b][c] = ???
Główna wartość tej klasy to funkcja GetOrCreate(var key)